bcdaworker/worker/worker_test.go

```diff
@@ -282,11 +292,6 @@ func (s *WorkerTestSuite) TestWriteEOBDataToFileWithErrorsBelowFailureThreshold(
 	fData = fData[:len(fData)-1]
 	assertEqualErrorFiles(s.T(), ooResp, string(fData))
 
-	// Assert cmsID and jobID fields are being added to the logs
-	assert.Equal(s.T(), 2, len(logHook.Entries))
```

Comment:
these were removed because we no longer add `cmsID` and `jobID` in the `writeBBDataToFile` function